DylibBinder/DBParameter.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using SwiftReflector;
+
+namespace DylibBinder {
+	internal class DBParameter {
+		public DBParameter (string type, string publicName, string privateName, int index, ParameterOptions parameterOptions)
+		{
+			Type = type;
+			PublicName = publicName;
+			PrivateName = privateName;
+			Index = index;
+			IsVariadic = parameterOptions.HasFlag (ParameterOptions.IsVariadic);
+			HasInstance = parameterOptions.HasFlag (ParameterOptions.HasInstance);
+			IsConstructor = parameterOptions.HasFlag (ParameterOptions.IsConstructor);
+			IsEmptyParameter = parameterOptions.HasFlag (ParameterOptions.IsEmptyParameter);
```

Comment:
Properties are not public variables, they can be calculated on the fly, meaning, no need to set a property for that. store the parameterOptions in a variable, then write the properties as:
```csharp
public bool IsConstructor => parameterOptions.HasFlag (ParameterOptions.IsConstructor);
```

Reply:
Im not sure I understand this one fully